devices/xiaomi.js

```diff
@@ -98,6 +114,87 @@ const fzLocal = {
             return result;
         },
     },
+    aqara_feeder: {
+        cluster: 'aqaraOpple',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            Object.entries(msg.data).forEach(([key, value]) => {
+                switch (parseInt(key)) {
+                case 0xfff1: {
+                    const attr = value.slice(3, 7);
+                    const len = value.slice(7, 8).readUInt8();
+                    const val = value.slice(8, 8 + len);
+                    switch (attr.readInt32BE()) {
+                    case 0x04150055: // feeding
+                        result['feed'] = '';
+                        break;
+                    case 0x041502bc: { // feeding report
+                        const report = val.toString();
+                        result['feeding_source'] = {1: 'manual', 2: 'remote'}[parseInt(report.slice(0, 2))];
+                        result['feeding_size'] = parseInt(report.slice(3, 4));
+                        break;
+                    }
+                    case 0x0d680055: // portions per day
+                        result['portions_per_day'] = val.readUInt16BE();
+                        break;
+                    case 0x0d690055: // weight per day
+                        result['weight_per_day'] = val.readUInt32BE();
+                        break;
+                    case 0x0d0b0055: // error ?
+                        result['error'] = {1: true, 0: false}[val.readUInt8()];
+                        break;
+                    case 0x080008c8: { // schedule string
+                        const schlist = val.toString().split(',');
+                        const schedule = [];
+                        schlist.forEach((str) => { // 7f13000100
+                            const feedtime = Buffer.from(str, 'hex');
+                            schedule.push({
+                                'days': daysLookup[feedtime[0]],
+                                'hour': feedtime[1],
+                                'minute': feedtime[2],
+                                'size': feedtime[3],
+                            });
+                        });
+                        result['schedule'] = schedule;
+                        break;
+                    }
+                    case 0x04170055: // indicator
+                        result['led_indicator'] = {1: 'ON', 0: 'OFF'}[val.readUInt8()];
```

Comment:
I believe simple ternary operator would be more readable
`val.readUInt8() == 1 ? 'ON' : 'OFF'`

Reply:
not critical, but looks uniform to other attributes